Jenkinsfile

```diff
@@ -1,131 +1,174 @@
 #!/bin/env groovy
+@Library('cliqz-shared-library@v1.2') _
+def matrix = [
+        'cliqz':[
+            'config':'mozconfigs/cliqz.mozconfig',
+            'test': true, 
+        ],
+        'ghostery':[
+            'config':'mozconfigs/ghostery.mozconfig',
+            'test': false,
+        ],
+        'cliqz-alpha':[
+            'config':'mozconfigs/cliqz-alpha.mozconfig',
+            'test': false,
+        ],
+        'ghostery-alpha':[
+            'config':'mozconfigs/ghostery-alpha.mozconfig',
+            'test': false,
+        ],
+    ]
 
-node('us-east-1 && ubuntu && docker && !gpu') {
-
-    def dockerTag = ""
-    def apk = ""
-    try {
-        withGenymotion(
-            "ami-13050368",
-            "1",
-            "t2.medium",
-            "android_ci_genymotion",
-            "sg-5bbf173f",
-            "subnet-341ff61f",
-            "us-east-1"
-            ) {
+def build(Map m){
+    def flavor = m.config
+    def flavorname = m.name
+    def nodeLabel = 'us-east-1 && ubuntu && docker && !gpu'
+    def test =m.test
+    return {
+        node(nodeLabel){
+            def dockerTag = ""
+            def apk = ""
             stage('Checkout') {
                 checkout scm
                 dockerTag = readFile('mozilla-release/browser/config/version_display.txt').trim()
-                withCredentials([file(credentialsId: 'ceb2d5e9-fc88-418f-aa65-ce0e0d2a7ea1', variable: 'SSH_KEY')]) {
-                    cloneRepoViaSSH(
-                        "git@github.com:cliqz/autobots.git",
-                        "-b version2.0 --single-branch --depth=1"
-                    )
-                }
             }
             def baseImageName = "browser-f/android:${dockerTag}"
             docker.withRegistry('https://141047255820.dkr.ecr.us-east-1.amazonaws.com') {
                 docker.image("${baseImageName}").inside {
-                    stage('Build APK') {
-                        sh '''#!/bin/bash -l
+                    stage('Build APKS') {
+                        sh """#!/bin/bash -l
                             set -e
                             set -x
-                            cp mozconfigs/cliqz.mozconfig mozilla-release/mozconfig
+                            rm -rf autobot*
+                            cp ${flavor} mozilla-release/mozconfig
                             cd mozilla-release
+                            ./mach clobber
                             ./mach build
-                            for lang in `ls ../l10n/`; do
```

Comment:
Why did you remove this? This is mandatory for multi-language builds

Reply:
Will put it back